### PR TITLE
Issue/#164 update the frontend readme

### PR DESCRIPTION
**Pull request by MarioSinovcic**
- [x] The pull request is complete according to the following criteria:
  - [x] Acceptance criteria have been met
  - [x] The documentation is kept up-to-date
  - [x] Comprehensive tests (if applicable) have been generated and all pass.
  - [x] The pull request describes the changes that have been made, and enough information is present in the description for any developer to understand what has changed
  - [x] Commits have been squashed (or will be on merge).
  - [x] The branch name is descriptive and follows the pull request title format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [x] The pull request title is of the following format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [x] The description uses [github syntax](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to link to the issue. E,g Resolves se701g2/Doto#{Number}
  - [x] At least two reviewers assigned. One of which must be the assigner of the issue.
  - [x] If there are merge conflicts, run git rebase as opposed to git merge with master.

Resolves se701g2/Doto#164

### CODE REVIEW

File: doto-frontend/README.md [hunk 1]
```diff
@@ -37,6 +37,84 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+    <Route path="/settings">
+      <ThemeContext.Provider value={[theme, setTheme]}>
+        <SettingsPage />
+      </ThemeContext.Provider>
+    </Route>
+```
+
+New pages that are added should following the same routing structure setup in `Route.js`.
+
+More documenation [here](https://reacttraining.com/react-router/web/guides/quick-start)
```
Comment (res550) [inline doto-frontend/README.md]: Spelling error here for documenation should be "documentation"

File: doto-frontend/README.md [hunk 2]
```diff
@@ -37,6 +37,84 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+  className="text-area spacing"
+  id="standard-basic"
+  label="Location"
+  onChange={handleLocationChange}
+/>
+```
+
+Where possible, material ui components should be adopted and implemented according to the provided documentation.
+
+More documenation [here](https://material-ui.com/)
```
Comment (res550) [inline doto-frontend/README.md]: Spelling error here for documenation should be "documentation"

File: doto-frontend/README.md [hunk 3]
```diff
@@ -37,6 +37,84 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+            <Route />
+        </MemoryRouter>,
+    );
+    expect(wrapper.find(SettingsPage)).toHaveLength(1);
+});
+```
+
+All new components should include relevant tests using the thes two toolchains.
+
+More documenation [here](https://jestjs.io/) & [here](https://enzymejs.github.io/enzyme/)
```
Comment (res550) [inline doto-frontend/README.md]: Spelling error here for documenation should be "documentation"

File: doto-frontend/README.md [hunk 4]
```diff
@@ -37,6 +37,84 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+Implementation example:
+
+```
+<h2 class="text-lg">Calendar</h2>
+<div class="text-purple-500">Support</div>
+```
+
+The root level `tailwind.js` file contains a list of the different CSS variants.
+
+More documenation [here](https://tailwindcss.com/)
```
Comment (res550) [inline doto-frontend/README.md]: Spelling error here for documenation should be "documentation"

**Comment by MarioSinovcic** [comment]
@res550 I appreciate you finding those mistakes, they should all be fixed now 😄

**Pull request merged**